great-idea/css/index.css

```diff
@@ -63,4 +63,168 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+nav {
```

Comment:
You could add display: flex on the nav and then align items: center and that will center everything in the nav vertically